### PR TITLE
feat(core): expose `TuiNotificationDefaultOptions`

### PR DESCRIPTION
**Pull request by splincode**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring
- [ ] Code style update
- [ ] Build or CI related changes
- [ ] Documentation content changes

### CODE REVIEW

File: projects/demo/src/modules/services/notifications/notifications.component.ts [hunk 1]
```diff
@@ -50,6 +50,7 @@ import {AlertExampleWithDataComponent} from './examples/4/alert-example-with-dat
     changeDetection,
 })
 export class ExampleTuiNotificationsComponent {
+    readonly method = import('!!raw-loader!./method.md');
     readonly exampleImportModuleComponent = exampleImportModuleComponent;
```
Comment (waterplea) [inline projects/demo/src/modules/services/notifications/notifications.component.ts:54]: Let's switch this whole file to `import` and `md`

**Review by waterplea — APPROVED** [review]

File: projects/demo/src/modules/services/notifications/notifications.component.ts (hunk 1 above)
Comment (splincode) [reply projects/demo/src/modules/services/notifications/notifications.component.ts]: fixed

**Comment by splincode** [comment]
@nsbarsukov @vladimirpotekhin hello) please review me

File: projects/core/tokens/notification-options.ts [hunk 2]
```diff
@@ -1,29 +1,45 @@
...
     readonly status: TuiNotification;
     readonly hasIcon: boolean;
     readonly hasCloseButton: boolean;
+    readonly defaultAutoCloseTime: number;
 }
 
+/**
+ * @deprecated: remove in v3.0, use TuiNotificationDefaultOptions
+ */
+export type NotificationTokenOptions = TuiNotificationDefaultOptions;
```
Comment (nsbarsukov) [inline projects/core/tokens/notification-options.ts:19]: Do we need word `default` ?

I mean
```suggestion
export type NotificationTokenOptions = TuiNotificationOptions;
```

Other similar options dont have it in their naming.
Example
https://github.com/Tinkoff/taiga-ui/blob/bd5c1fbd7b8b6672e22a459bf13ef9d34f7923ba/projects/addon-commerce/components/money/money-options.ts#L6

File: projects/demo/src/modules/services/notifications/notifications.template.html [hunk 3]
```diff
@@ -57,21 +57,8 @@
...
-                    </div>
-                    <div>
-                        &nbsp;&nbsp;&nbsp;&nbsp;options: TuiNotificationOptions
-                        | TuiNotificationOptionsWithData&lt;I&gt;
-                    </div>
-                    <div>): Observable&lt;O&gt;</div>
-                </code>
-            </p>
+            <tui-doc-code [code]="method"></tui-doc-code>
+            <br />
```
Comment (nsbarsukov) [inline projects/demo/src/modules/services/notifications/notifications.template.html:61]: https://github.com/Tinkoff/taiga-ui/pull/1184#discussion_r786488101

Comment (nsbarsukov) [reply projects/demo/src/modules/services/notifications/notifications.template.html:61]: use class `tui-space_bottom-*` if you want to make empty space below

File: projects/core/tokens/notification-options.ts (hunk 2 above)
Comment (splincode) [reply projects/core/tokens/notification-options.ts:19]: I have problem with existing interface

https://github.com/Tinkoff/taiga-ui/blob/main/projects%2Fcore%2Finterfaces%2Fnotification-options.ts

**Review by nsbarsukov — APPROVED** [review]

File: projects/demo/src/modules/services/notifications/notifications.template.html (hunk 3 above)
Comment (splincode) [reply projects/demo/src/modules/services/notifications/notifications.template.html]: fixed

**Pull request merged**